packages/roosterjs-content-model-dom/lib/modelApi/editing/retrieveModelFormatState.ts

```diff
@@ -241,14 +252,20 @@ function mergeValue<K extends keyof ContentModelFormatState>(
     key: K,
     newValue: ContentModelFormatState[K] | undefined,
     isFirst: boolean,
-    parseFn: (val: ContentModelFormatState[K]) => ContentModelFormatState[K] = val => val
+    parseFn: (val: ContentModelFormatState[K]) => ContentModelFormatState[K] = val => val,
+    callbacks?: MergeFormatValueCallbacks
```

Comment:
It seems the parameter is always passed, so you can make it a required parameter, and put it before the optional parameter `parseFn`, so no need to always add `undefined` when call this function

Reply:
Updated, thanks!